### ISSUE TITLE
Docker Container is crashing after no response

### ISSUE THREAD

**Issue by Frooger** [benc-uk/postman-prometheus#9]
Hi I use a simple postman collection for requesting endpoints with your repo, btw nice !

I run your repo as an docker container

[config]:
```
...
    environment:
      COLLECTION_FILE: /collection.json
      RUN_INTERVAL: 300
...
```

Everything works fine until one endpoint give no response anymore, 
after that the container seems to crash because of the error Log 

[Log]:

```
/runner/server.js:176
      ` - Completed request '${summary.run.executions[e].item.name}' in ${summary.run.executions[e].response.responseTime} ms`
                                                                                                             ^

TypeError: Cannot read properties of undefined (reading 'responseTime')
    at runComplete (/runner/server.js:176:110)
    at /node_modules/newman/lib/run/index.js:346:25
    at wrapper (/node_modules/async/internal/once.js:12:16)
    at eachOfArrayLike (/node_modules/async/eachOf.js:45:9)
    at eachOf (/node_modules/async/eachOf.js:181:12)
    at awaitable (/node_modules/async/internal/awaitify.js:13:28)
    at eachLimit (/node_modules/async/each.js:125:31)
    at awaitable (/node_modules/async/internal/awaitify.js:13:28)
    at done (/node_modules/newman/lib/run/index.js:343:21)
    at /node_modules/postman-runtime/lib/backpack/index.js:58:34
    at PostmanCollectionRun._process (/node_modules/postman-runtime/lib/runner/run.js:163:13)
    at PostmanCollectionRun.<anonymous> (/node_modules/postman-runtime/lib/runner/run.js:169:76)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)
```

Seems to be here

https://github.com/benc-uk/postman-prometheus/blob/ece229c9beaafa6efa91bd9dc208fccdb838c03c/src/server.js#L176

**Comment by Frooger** [comment]
i made an pull request for this issue :)

#10

**Comment by Bak3y** [comment]
Updated PR including your fix and a linter fix https://github.com/benc-uk/postman-prometheus/pull/15

**Comment by Bak3y** [comment]
Should be fixed with #15 merge @Frooger heads up
